.github/workflows/mac_binary.yml

```diff
@@ -1,26 +0,0 @@
-name: MacOS binary release
-on:
-  workflow_dispatch:
-  push:
-    branches:
-      - master
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]+*'
-jobs:
-  build_binary:
-    runs-on: macOS-latest
-
-    steps:
-    - uses: hecrj/setup-rust-action@v1
-      with:
-        rust-version: nightly-2020-03-19
```

Comment:
Would it be sufficient to update this instead? 
I'm totally fine with dropping it all together as well.

Reply:
Could be, but I also wonder if we should support architecture which is abandoned for 3.5 years already.

Reply:
Ah, didn't realize that, yeah let's do it.